docs/core/build/distribution-packaging.md

```diff
@@ -124,38 +126,51 @@ The following lists the recommended packages:
   - **Version:** \<runtime version>
   - **Example:** dotnet-runtime-2.1
   - **Contains:** (5)
-  - **Dependencies:** `dotnet-hostfxr:<runtime version>+`, `dotnet-runtime-deps-[major].[minor]`
+  - **Dependencies:** `dotnet-hostfxr-[major].[minor]`, `dotnet-runtime-deps-[major].[minor]`
+  - **Provides shared directory:** `{dotnet_root}/shared`, `{dotnet_root}/shared/Microsoft.NETCore.App`
 
-- `dotnet-hostfxr` - dependency
+- `dotnet-hostfxr-[major].[minor]` - dependency
   - **Version:** \<runtime version>
-  - **Example:** dotnet-hostfxr
+  - **Example:** dotnet-hostfxr-3.0
   - **Contains:** (2)
-  - **Dependencies:** `host:<runtime version>+`
+  - **Dependencies:** `dotnet-host`
+  - **Provides shared directory:** `{dotnet_root}/host/fxr`
 
 - `dotnet-host` - dependency
   - **Version:** \<runtime version>
   - **Example:** dotnet-host
   - **Contains:** (1),(8),(9),(10),(16)
+  - **Provides shared directory:** `{dotnet_root}`, `{dotnet_root}/host`
 
 - `dotnet-apphost-pack-[major].[minor]` - dependency
   - **Version:** \<runtime version>
   - **Contains:** (13)
+  - **Dependencies:** `dotnet-host` (provides `{dotnet_root}`)
+  - **Provides shared directory:** `{dotnet_root}/packs`, `{dotnet_root}/packs/Microsoft.NETCore.App.Host.<rid>`
 
 - `dotnet-targeting-pack-[major].[minor]` - Allows targeting a non-latest runtime
   - **Version:** \<runtime version>
   - **Contains:** (12)
+  - **Dependencies:** `dotnet-host` (provides `{dotnet_root}`)
+  - **Provides shared directory:** `{dotnet_root}/packs`, `{dotnet_root}/packs/Microsoft.NETCore.App.Ref`
 
 - `aspnetcore-targeting-pack-[major].[minor]` - Allows targeting a non-latest runtime
   - **Version:** \<aspnetcore runtime version>
   - **Contains:** (11)
+  - **Dependencies:** `dotnet-host` (provides `{dotnet_root}`)
+  - **Provides shared directory:** `{dotnet_root}/packs`, `{dotnet_root}/packs/Microsoft.AspNetCore.App.Ref`
 
 - `netstandard-targeting-pack-[major].[minor]` - Allows targeting a netstandard version
   - **Version:** \<sdk version>
   - **Contains:** (15)
+  - **Dependencies:** `dotnet-host` (provides `{dotnet_root}`)
+  - **Provides shared directory:** `{dotnet_root}/packs`, `{dotnet_root}/packs/NETStandard.Library.Ref`
 
 - `dotnet-templates-[major].[minor]`
   - **Version:** \<sdk version>
   - **Contains:** (15)
+  - **Dependencies:** `dotnet-host` (provides `{dotnet_root}`)
+  - **Provides shared directory:** `{dotnet_root}/templates`
```

Comment:
I don't have a strong preference, but as an alternative to the 

> **Dependencies:** `dotnet-host` (provides `{dotnet_root}`)

We could add `{dotnet_root}` to the **Provides shared directory:**.

@omajid @dagood do you have some thoughts about this?

Reply:
My 2 cents: Maybe this is too much detail? I am not sure if something like "provides shared directory" needs to be completely specified.

Reply:
I would leave it out, given that it doesn't apply to all packaging systems.

Packagers are free to deviate from this doc when it makes sense, right?